### PR TITLE
Honor password logging

### PR DESCRIPTION
**Pull request by martinvanhensbergen**
The FTP policy allows adding extra commands to be logged, including the PASS command. When doing so, the actual plaintext password will be logged as argument of the command even though password_capture may be set to F.

With this change, if password capture is false, the password as part of the PASS argument will also be hidden.

### CODE REVIEW

File: scripts/base/protocols/ftp/main.zeek
Comment (awelzel) [inline scripts/base/protocols/ftp/main.zeek:174]: You could condense to this to:
```
return ! s$capture_password && to_lower(s$user) !in guest_ids;
```

File: scripts/base/protocols/ftp/main.zeek
Comment (awelzel) [inline scripts/base/protocols/ftp/main.zeek:185]:
```suggestion
	local password_hidden_string = "<hidden>";
```

File: scripts/base/protocols/ftp/main.zeek
Comment (awelzel) [inline scripts/base/protocols/ftp/main.zeek:209]: Suggest to use a single line and remove the braces. Also below.

**Review by awelzel — CHANGES_REQUESTED** [review]
Hey @martinvanhensbergen - thanks!

Have only nits - mind taking a look?

Would you be up for squashing some of the "Fix" commits before a merge? Otherwise can do so, too.

**Comment by martinvanhensbergen** [comment]
Hi @awelzel  thanks for reviewing and the suggested changes. I will make the changes and squash and cleanup a little later this week!

**Comment by martinvanhensbergen** [comment]
Hi @awelzel 

I have applied the changes in the code as you suggested. I was wondering if squashing is necessary as i'd say this is done automatically by github on merge?

Regards
Martin

**Comment by awelzel** [comment]
@martinvanhensbergen  - thanks!
> I have applied the changes in the code as you suggested. I was wondering if squashing is necessary as i'd say this is done automatically by github on merge?

We don't use the GitHub merge button :-) If you don't want to squash yourself, can also do during the merge by hand.

**Comment by martinvanhensbergen** [comment]
If you would be able to do the merging & squash then that would be much appreciated ❤️ Thanks for the code suggestions though; it looks a bit cleaner indeed :)

**Comment by awelzel** [comment]
Merged with c36ae2f1a3e572676456dfb1230d776b9d0631bc.

**Pull request closed without merging**
